webapp/src/components/common/dialogs/ImportDialog.tsx

```diff
@@ -44,31 +62,35 @@ function ImportDialog(props: Props) {
   // Event Handlers
   ////////////////////////////////////////////////////////////////
 
-  const handleDrop = async (acceptedFiles: File[]) => {
-    const fileToUpload = R.last(acceptedFiles);
-
-    if (fileToUpload) {
-      setIsUploading(true);
-      setUploadProgress(0);
-
-      try {
-        await onImport(fileToUpload, (progress) => {
-          if (isMounted()) {
-            setUploadProgress(progress);
-          }
-        });
-
-        if (isMounted()) {
-          onClose();
-        }
-      } catch {
-        // noop
-      } finally {
-        if (isMounted()) {
-          setIsUploading(false);
-          setUploadProgress(-1);
-        }
-      }
+  async function handleDropAccepted(acceptedFiles: File[]) {
+    setInvalidText("");
```

Comment:
if used to display error messages maybe `setError`

Reply:
I want to make a distinction between submit error (server side) and invalid value (front side) (like for field validation).